temi_code/curate_my_heart_counts_public_BRIDGE.R

```diff
@@ -3,183 +3,275 @@ library(tidyverse)
 library(lubridate)
 library(zipcode)
 data(zipcode)
-# a comment
 
+#takes in table data from synapse and stores relevant columns
 read_syn_table <- function(syn_id) {
   q <- synTableQuery(paste('select "healthCode","recordId","createdOn",',
                            '"appVersion","phoneInfo"',
                            'from', syn_id))
-  return(q$asDataFrame())
-  print(q$asDataFrame())
+  return(q$asDataFrame()) #stores synapse table data as a dataframe within R
 }
 
+#takes in dataframe, adds two new columns (id, activity), excludes two existing columns (row_id, row_version)
 curate_table <- function(synId, activity_name) {
   df <- read_syn_table(synId) %>%
-        select(-ROW_ID, -ROW_VERSION) %>%
-        mutate(activity = activity_name,
+        select(-ROW_ID, -ROW_VERSION) %>% #excludes Row_id and Row_version from given dataframe 
```

Comment:
I haven't tried - are you sure you can add in-line comments within a `%>%` operator?

Reply:
yeah it seems to run fine despite that operator, but I am not too sure about within a function, that seems a little problematic, though I haven't had any problems with that yet